packages/@aws-cdk/aws-rds/lib/props.ts

```diff
@@ -84,6 +84,12 @@ export interface InstanceProps {
    * @default - true
    */
   readonly deleteAutomatedBackups?: boolean;
+  /**
+   * Indicates whether the DB instance is an internet-facing instance
+   *
+   * @default - `true` if the instance is deployed in a `PUBLIC` subnet, `false` otherwise.
```

Comment:
Let's make it exactly precise what happens here:

```suggestion
   * @default - true if vpcSubnets is `SubnetType.PUBLIC`, false otherwise
```

Same comment for the Instance property.